helm_deploy/hmpps-electronic-monitoring-datastore-api/values.yaml

```diff
@@ -2,6 +2,8 @@ generic-service:
   nameOverride: hmpps-electronic-monitoring-datastore-api
   productId: "HMPPS529" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
 
+  service-account-name: hmpps-electronic-monitoring-datastore
```

Comment:
As discussed: this will need to be removed, and will be overwritten by the env-specific values, e.g. dev here:
https://github.com/ministryofjustice/hmpps-electronic-monitoring-datastore-api/blob/4d941b7f79061ce33a2ef2066684a7f517b06454/helm_deploy/values-dev.yaml#L7C24-L7C53